config/import-control-test.xml

```diff
@@ -3,7 +3,9 @@
     "-//Checkstyle//DTD ImportControl Configuration 1.4//EN"
     "https://checkstyle.org/dtds/import_control_1_4.dtd">
 
-<import-control pkg="com.puppycrawl.tools.checkstyle">
+<import-control
+        pkg="(com\.puppycrawl\.tools\.checkstyle|com\.google\.checkstyle\.test|com\.sun\.checkstyle\.test|org\.checkstyle.*)"
```

Comment:
@rnveach 
I suggested @shashwatj07 to have full package name in regexp, it can be shortened but will be clear